models/spring-ai-anthropic/src/main/java/org/springframework/ai/anthropic/AnthropicChatModel.java

```
@@ -355,6 +357,16 @@ else if (mediaData instanceof String text) {
 
 	}
 
+	private Type getContentBlockTypeByMedia(Media media) {
+		if (media.getMimeType().toString().startsWith("image")) {
+			return Type.IMAGE;
+		}
+		else if (media.getMimeType().toString().contains("pdf")) {
+			return Type.DOCUMENT;
+		}
+		return Type.IMAGE;
```

if it wasn't classified as image, prob better to throw an exception